helmclient.go

```diff
@@ -660,6 +664,8 @@ func (c *Client) UpdateReleaseFromTarball(ctx context.Context, releaseName, path
 		release, err := c.newHelmClientFromTunnel(t).UpdateRelease(releaseName, path, options...)
 		if IsReleaseNotFound(err) {
 			return backoff.Permanent(microerror.Mask(err))
+		} else if IsYamlConvertingFailed(err) {
```

Comment:
To mitigate the converting issue stop making other CR reconciliation, I add this new error checking function.